### PR TITLE
[Validator] Add `EnableAutoMapping` and `DisableAutoMapping` constraints

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/12563

### CODE REVIEW

File: reference/constraints/DisableAutoMapping.rst [hunk 1]
```diff
@@ -0,0 +1,119 @@
+DisableAutoMapping
+==================
+
+This constraint allows to disable Doctrine's ``auto_mapping`` on a class or a
+property. You can read more about it
+:ref:`here <doctrine_auto-mapping>`. Automapping allows to determine
+validations rules based on Doctrine's annotations and attributes. You may
```
Comment (xabbuh) [inline reference/constraints/DisableAutoMapping.rst]:
```suggestion
validation rules based on Doctrine's annotations and attributes. You may
```

File: reference/constraints/DisableAutoMapping.rst [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+Applies to  :ref:`property or method <validation-property-target>`
+Class       :class:`Symfony\\Component\\Validator\\Constraints\\DisableAutoMapping`
+==========  ===================================================================
+
+Basic Usage
+-----------
+
+In the following example, the
+:class:`Symfony\\Component\\Validator\\Constraints\\DisableAutoMapping`
+constraint will tell the validator to gather constraints from Doctrine's
```
Comment (xabbuh) [inline reference/constraints/DisableAutoMapping.rst]:
```suggestion
constraint will tell the validator to not gather constraints from Doctrine's
```

File: reference/constraints/EnableAutoMapping.rst [hunk 3]
```diff
@@ -0,0 +1,119 @@
+EnableAutoMapping
+=================
+
+This constraint allows to enable Doctrine's ``auto_mapping`` on a class or a
+property. You can read more about it
+:ref:`here <doctrine_auto-mapping>`. Automapping allows to determine
+validations rules based on Doctrine's annotations and attributes. You may
```
Comment (xabbuh) [inline reference/constraints/EnableAutoMapping.rst]:
```suggestion
validation rules based on Doctrine's annotations and attributes. You may
```

**Review by xabbuh — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Alex. I didn't know that we were missing this. While merging in upper branches, I removed the example with annotations.

**Pull request merged**